### ISSUE TITLE
Enable e2edev service suspension test for policy case

### ISSUE THREAD

**Issue by linggao** [open-horizon/anax#1038]

**Comment by diru1100** [comment]
can I work on this issue? @linggao

**Comment by linggao** [comment]
@diru1100 I have updated the description for this issue. Please it is not for `anax/policy/policy_file_test.go`. This file is for unit test. 
When we talk about e2edev testing, it is refer to "end to end testing". The testing files are under `anax/test/` directories.
Please let me or @joewxboy  know that you still want to work on this issue.

**Comment by diru1100** [comment]
@linggao
Yes, I am still interested. I find testing is a good way to understand a repo. Thank you for updating the issue.

**Comment by joewxboy** [comment]
@diru1100 I've assigned this to you.  Please reach out to @linggao or @dabooz with any questions, and when you're ready to submit a PR.

**Comment by diru1100** [comment]
I am trying to understand about e2edev test and how to write one. I went through some of the previous meetings of QA WG and read about policies and patterns. 

Is the command to test: ``` make test TEST_VARS="NOLOOP=1 TEST_PATTERNS=sall"```?
As I am unable to run the command you mentioned in the ```anax/test ```folder. When I run it from``` anax/``` it is not running the tests. 

What I understood so far is :
- gov folder has all the e2edev test files.
- gov-combined.sh does all the routing on which tests to be run?
- I thought of just changing the if condition in the main code i.e to ``` if [ "${PATTERN}" != "" && "${PATTERN}" != "sall" ]; then``` 

Any more resources or advice would be appreciated @linggao @dabooz

**Comment by linggao** [comment]
@diru1100
I usually run the e2edev under `anax/test` directory.
For the pattern case that do not include the K8S testing:
    `TEST_VARS="NOLOOP=1  NOK8S=1 NOKUBE=1 TEST_PATTERNS=sall" make test `
For the policy case that do not include k8S testing:
    `TEST_VARS="NOLOOP=1  NOK8S=1 NOKUBE=1" make test `

**Comment by LiilyZhang** [comment]
Close as complete
